Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -484,6 +556,12 @@ namespace O3DE::ProjectManager
         m_projectInfo.m_logUrl = logUrl;
     }
 
+    void ProjectButton::SetProgressBarPercentage(const float percent)
+    {
+        m_projectImageLabel->GetProgressBar()->setValue(static_cast<int>(percent*100));
+        m_projectImageLabel->GetProgressPercentage()->setText(QString("%1%").arg(static_cast<int>(percent*100)));
+    }
```

Comment:
This could be updated to accept a double for better precision on the percent value